plugins/chat/assets/javascripts/discourse/components/chat/drawer-routes/channel.gjs

```diff
@@ -12,6 +12,14 @@ export default class ChatDrawerRoutesChannel extends Component {
   @service chatStateManager;
   @service chatChannelsManager;
 
+  get backBtnRoute() {
+    if (this.chat.activeChannel?.isDirectMessageChannel) {
```

Comment:
One day we need to get rid of this activeChannel thing and pass the objects through the custom routing